pandas-stubs/core/arrays/base.pyi

```diff
@@ -42,8 +42,9 @@ class ExtensionArray:
     def shift(self, periods: int = ..., fill_value: object = ...) -> Self: ...
     def unique(self): ...
     def searchsorted(self, value, side: str = ..., sorter=...): ...
-    # TODO: remove keyword-only when pandas removed na_sentinel
-    def factorize(self, *, use_na_sentinel: bool = ...) -> tuple[np.ndarray, Self]: ...
+    def factorize(
+        self, sort: bool = ..., use_na_sentinel: bool = ...
```

Comment:
Looking at the docs and the code, for `ExtensionArray.factorize()`, there is no `sort` argument, so remove that.

Reply:
Indeed this was my mistake, removed.